CONTRIBUTING.md

```diff
@@ -1,35 +1,50 @@
 # Contributing to Sentry SDK for Python
 
-We welcome contributions to python-sentry by the community. See the [Contributing to Docs](https://docs.sentry.io/contributing/) page if you want to fix or update the documentation on the website.
+We welcome contributions to `sentry-python` by the community.
 
-## How to report a problem
+This file outlines the process to contribute to the SDK itself. For contributing to the documentation, please see the [Contributing to Docs](https://docs.sentry.io/contributing/) page.
+
+## How to Report a Problem
 
 Please search the [issue tracker](https://github.com/getsentry/sentry-python/issues) before creating a new issue (a problem or an improvement request). Please also ask in our [Sentry Community on Discord](https://discord.com/invite/Ww9hbqr) before submitting a new issue. There is a ton of great people in our Discord community ready to help you!
 
 If you feel that you can fix or implement it yourself, please read a few paragraphs below to learn how to submit your changes.
 
-## Submitting changes
+## Submitting Changes
 
-- Setup the development environment.
-- Clone sentry-python and prepare necessary changes.
+- Fork the `sentry-python` repo and prepare your changes.
 - Add tests for your changes to `tests/`.
 - Run tests and make sure all of them pass.
 - Submit a pull request, referencing any issues it addresses.
 
-We will review your pull request as soon as possible.
-Thank you for contributing!
+We will review your pull request as soon as possible. Thank you for contributing!
+
+## Development Environment
+
+### Set up Python
+
+Make sure that you have Python 3 installed. Version 3.7 or higher is required to run style checkers on pre-commit.
+
+On macOS, we recommend using `brew` to install Python. For Windows, we recommend an official [python.org](https://www.python.org/downloads/) release.
 
-## Development environment
+### Fork the Repo
 
-### Clone the repo:
+Go to the [home page of the repo](https://github.com/getsentry/sentry-python), click the "Fork" button, and go through the process of creating your fork of `sentry-python`.
+
+### Clone the Forked Repo
+
+You should now have a new `sentry-python` repo in your GitHub projects. Clone it to work on it locally:
 
 ```bash
-git clone git@github.com:getsentry/sentry-python.git
+git clone git@github.com:<YOUR-USERNAME>/sentry-python.git
 ```
 
-Make sure that you have Python 3 installed. Version 3.7 or higher is required to run style checkers on pre-commit. On macOS, we recommend using brew to install Python. For Windows, we recommend an official python.org release.
+Replace `<YOUR-USERNAME>` with your actual GitHub username. Alternatively, you can go to your GitHub profile, find the forked `sentry-python` repo, and copy the clone URL from there.
```

Comment:
I suggest deleting these lines. Most developers will know how to clone a repository, and those who don't can easily find instructions for doing so in GitHub. Including these lines provides little benefit in my opinion, since the bash code line is not copy-pasteable as-is, since people will need to enter their username, and not everyone has SSH authentication for GitHub, which the provided command relies on.